src/style.css

```diff
@@ -1,45 +1,75 @@
+/* Basic styling */
 * {margin: 0; padding: 0; border: 0; vertical-align: baseline;}
 html {box-sizing: border-box;}
 *, *:before, *:after {box-sizing: inherit;}
-
-html, body {
-    height: 100%;
-    margin: 0;
-  }
+html, body {height: 100%; margin: 0;}
 
 body {
   font-family: "Segoe UI", sans-serif;
   background: #1A2933;
   color: #FFF;
 }
+
+h1 {
+  margin: 0 0 10px 0;
+  font-weight: 600;
+  line-height: 1.2;
+}
+
+p {
+  margin-top: 10px;
+  color: rgba(255,255,255,0.4);
+}
+
+/* Styling of window frame and titlebar */
 body {
   border: 1px solid #48545c;
   overflow-y: hidden;
 }
 
-
+body.maximized {
+  border: none;
+}
```

Comment:
I noticed this border doesn't show up when the window has a maximized state. In #6 / #7 the border shows when the maximize button is clicked because the bug means the window isn't actually given the maximized state, even though it takes up the whole screen.

Anyway, no harm in accepting this change.

Reply:
Update: actually it seems in the maximized state, Electron "consumes" 1px around the window (i.e. a 1px high div at the top will disappear), so best to leave the border in and let it get consumed

Reply:
Wow it seems my electron behaves slightly differently from others. I know what you mean, but absolutely no consuming happens to me. Weird.

Reply:
![image](https://user-images.githubusercontent.com/20509848/68165645-126f7000-ffb4-11e9-99fb-004a86fbd7ef.png)
![image](https://user-images.githubusercontent.com/20509848/68165667-1f8c5f00-ffb4-11e9-93b3-5ada53dc5ded.png)

Absolutely nothing

Reply:
Interesting, I imagine it's because you said you were on Electron 7.0.0... Maybe not consuming the border is not part of bug #6, but the new default behaviour. But yeah I still don't know why you're not affected by the bug. I might try updating to Electron 7.0.0 again to test it

Reply:
Also mine has this bug where when the window is maximized Electron says it's X & Y is -8px and it's width and height is +16px larger than it actually is.

Reply:
I tried with Electron 7.0.0 and the bug is still happening for me. But when the maximize property is successfully applied, e.g. by dragging the title bar up to the top of the screen, the 1px border is still present. Whether that's intended or a separate issue (or part of the same wider issue maybe), I don't know

Reply:
> Also mine has this bug where when the window is maximized Electron says it's X & Y is -8px and it's width and height is +16px larger than it actually is.

With Electron 5.0.12 `remote.getCurrentWindow().getBounds()` also says the same thing